srcpkgs/flintlib/update

```diff
@@ -0,0 +1,2 @@
+pkgname=flint
```

Comment:
Maybe mention in commit that name was picked because of existing package named flint.

Reply:
made a comment in the `update` file itself, hope that's ok